Formula/ace.rb

```diff
@@ -3,6 +3,9 @@ class Ace < Formula
   homepage "https://www.dre.vanderbilt.edu/~schmidt/ACE.html"
   url "http://download.dre.vanderbilt.edu/previous_versions/ACE-6.4.0.tar.bz2"
   sha256 "6a7fd5f23f6a004bb21978767df4e36adea078cd344e45bab50266213586c001"
+  revision 1
+
+  depends_on "openssl" => :build
```

Comment:
Well.. No, I'm not sure.
For what I understood, starting from macOS 10.11 Apple removed the headers of ssl from `/usr/include`. So, if a library/executable is compiled and linked against openssl, it will work anyway on macOS because of the presence of the lib in `/usr/lib/libssl.dylib`. 
But, if you have to compile an openssl-powered application, then we need to obtain the headers.

Now, I'm not more able to replicate the issue I had some days ago, so I'll delete this dependency.